# LMTD correction factor method

The simplest method, the log-mean temperature difference correction factor method, is as follows:

Q=U*A*ΔTLM*Ft

Considering in a heat exchanger, inlet temperature of cold fluid, Tci=15°C and hot fluid, Thi=130°C whereas outlet temperature of cold fluid, Tco=85°C and hot fluid, Tho=110°C. Rate of heat flow in this heat exchanger is 1MW.

Calculate ΔTLM.

In [23]:
!pip install ht #installed a Python package named ht

^C


Knowing the outlet and inlet temperatures of a heat exchanger and Q, one could determine UA as follows:

In [2]:
from ht.vectorized import LMTD, F_LMTD_Fakheri

dTlm = LMTD(Tci=15, Tco=85, Thi=130, Tho=110)
Ft = F_LMTD_Fakheri(Tci=15, Tco=85, Thi=130, Tho=110, shells=1)
Q = 1E6  # 1 MW
UA = Q / (dTlm * Ft)

In [31]:
F_LMTD_Fakheri(Tci=15, Tco=85, Thi=130, Tho=110, shells=1)

array(0.94383588)

In [33]:
LMTD(Thi=130, Tho=110, Tci=15, Tco=85)

array(66.91519847)

In [34]:
UA

15833.566307803789

Log mean temperature are available for both counterflow (by default) and co-current flow.

In [3]:
LMTD(Thi=130, Tho=110, Tci=15, Tco=85)

array(66.91519847)

In [4]:
LMTD(130, 110, 15, 85, counterflow=False)

array(58.97554356)

The difference in the LMTD values between counterflow (66.91519847) and parallel flow (58.97554356) configurations indicates that the heat transfer effectiveness can vary significantly based on the direction in which the fluids flow and exchange heat. Counterflow arrangements generally have a higher effectiveness compared to parallel flow arrangements for the same inlet and outlet temperatures of the fluids.

In chemical engineering applications, the Log Mean Temperature Difference (LMTD) method is fundamental for analyzing heat exchangers. It's used to determine the overall temperature difference between two fluids as they exchange heat in these systems. Engineers use LMTD to design and optimize heat exchangers, helping determine the required surface area for effective heat transfer between fluids.